javascript/javascript1/week4/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <script src="homeworkWeek4.js"></script>
```

Comment:
If you need to do the logical part before the content (html) and design(css) is loaded, you can place <script> inside <head>. 
But normally the content and design will be loaded first and only after you will go for the functionalities. So its much preferred to put the <script> inside the <body> and at the end of <body>